plugins/notifications/src/components/NotificationsTable/SeverityIcon.tsx

```diff
@@ -19,21 +19,39 @@ import NormalIcon from '@material-ui/icons/CheckOutlined';
 import CriticalIcon from '@material-ui/icons/ErrorOutline';
 import HighIcon from '@material-ui/icons/WarningOutlined';
 import LowIcon from '@material-ui/icons/InfoOutlined';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  critical: {
+    color: theme.palette.error.main,
+  },
+  high: {
+    color: theme.palette.warning.main,
+  },
+  normal: {
+    color: theme.palette.success.main,
+  },
+  low: {
+    color: theme.palette.primary.main,
```

Comment:
Realized this isn't a very good color to use, and some of the other ones too. You'll easily end up in a situation where integrators theme their instance in a way that breaks this. Perhaps the colors from `palette.status.*` work better?

Reply:
Done. The screenshot above is updated to conform the recent state.